src/Student1/Ex5/ShowHide.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import './ShowHide.scss'
+import React from 'react'
+
+export const ShowHide = (props) => {
+    let [show, setShow] = useState(false);
+    let [enableInput, setEnableInput] = useState(true);
```

Comment:
Try replacing this with the text that should be displayed on the button:  buttonText, setButtonText.